plugins/speculation-rules/uninstall.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Plugin uninstaller logic.
+ *
+ * @package speculation-rules
+ * @since 1.1.0
```

Comment:
Per the version you used elsewhere:
```suggestion
 * @since 1.2.0
```